bin/amphtml-update.py

```diff
@@ -772,6 +775,22 @@ def UnicodeEscape(string):
 	"""
 	return ('' + string).encode('unicode-escape')
 
+def AddMandatoryOf( spec_dict, attribute_spec, constraint ):
+	"""Gets the spec dictionary, with mandatory_*of constraints possibly added.
+
+	Args:
+		spec_dict: A dictionary of the attr_spec values to possibly add the constraint to.
+		attribute_spec: The attribute spec in which to look for the mandatory_*of constraint.
+		constraint: A string of the mandatory_*of constraint, like 'mandatory_anyof'.
+	Returns:
+		The spec_dict, with the constraint possibly added.
+	"""
+	if attribute_spec.HasField(constraint):
+		mandatory_of = getattr(attribute_spec, constraint).lstrip('[').rstrip(']').split(',')
+		spec_dict[constraint] = [oneof.strip(' ').strip("'") for oneof in mandatory_of]
```

Comment:
This will need to ensure that `'[src]'` becomes `'data-amp-bind-src'`, so replacing the first `[` with `data-amp-bind-` and stripping the last ']'.

Reply:
Ah, good point.

Reply:
Applied in https://github.com/ampproject/amp-wp/compare/39926b690632afdfe86458844d5e34614e85fb76...bcec1fa7e40d487b656f5dde699c650ef72a06cc